physionet-django/console/templates/console/event_management.html

```diff
@@ -76,4 +116,59 @@ <h5 class="modal-title">Participants</h5>
       </div>
     </div>
   </div>
+  <div class="modal fade"
+       id="pending_applications"
+       tabindex="-2"
+       role="dialog"
+       aria-labelledby="view-pending_applications-modal"
+       aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Pending Applications</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        {% include 'events/event_pending_applications.html' %}
+      </div>
+    </div>
+  </div>
+  <div class="modal fade"
+       id="rejected_applications"
+       tabindex="-1"
+       role="dialog"
+       aria-labelledby="view-rejected_applications-modal"
+       aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Rejected Applications</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        {% include 'events/event_rejected_applications.html' %}
+      </div>
+    </div>
+  </div>
+  <div class="modal fade"
+       id="withdrawn_applications"
+       tabindex="-1"
+       role="dialog"
+       aria-labelledby="view-withdrawn_applications-modal"
+       aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Withdrawn Applications</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        {% include 'events/event_withdrawn_applications.html' %}
+      </div>
+    </div>
+  </div>
+
```

Comment:
I see a lot of duplication in this snippet. Maybe introducing a variable like:
```python
[
  {
    "id": "withdrawn_applications",
    "title": "Withdrawn Applications",
    "template": "event_withdrawn_applications.html"
  },
  ...
]
```
and rendering that dynamically within a for-loop would be better.

Reply:
Updated the template and the view accordingly.